openapi-integration-tests/src/test/resources/ballerina_sources/bal_service_has_compilation_issue/service.bal

```diff
@@ -0,0 +1,10 @@
+import ballerina/http;
+
+service /cpi on new http:Listener(0) {
+    resource function get hello() {
+    }
+
+    resource function post path(@http:Payload string name) {
+    }
+    }
```

Comment:
Can you remove this additional bracket. I think we don't need it here..

Reply:
I think this bracket is added intentionally to produce a compiler error

Reply:
Got it thanks for the clarification!

Reply:
Yeah, it is for a negative test scenario for getting a compilation error.